pkg/api/types.go

```diff
@@ -527,6 +532,24 @@ type TestStepConfiguration struct {
 	OpenshiftInstallerCustomTestImageClusterTestConfiguration *OpenshiftInstallerCustomTestImageClusterTestConfiguration `json:"openshift_installer_custom_test_image,omitempty"`
 }
 
+type ClusterClaim struct {
+	// Product is the name of the product being released.
+	// Defaults to ocp.
+	Product ReleaseProduct `json:"product,omitempty"`
+	// Version is the version of the product
+	Version string `json:"version,omitempty"`
+	// Architecture is the architecture for the product.
+	// Defaults to amd64.
+	Architecture ReleaseArchitecture `json:"architecture,omitempty"`
+	// Cloud is the cloud where the product is installed, e.g., aws.
+	Cloud ClusterProfile `json:"cloud,omitempty"`
```

Comment:
I don't know that we necessarily want to use this type here

Reply:
I will define a new type for this.
But the change will be in https://github.com/openshift/ci-tools/pull/1838 and this PR will be rebased.